lib/libzfs/libzfs_sendrecv.c

```diff
@@ -4888,8 +4888,8 @@ zfs_receive_one(libzfs_handle_t *hdl, int infd, const char *tosnap,
 		zfs_nicebytes(bytes, buf1, sizeof (buf1));
 		zfs_nicebytes(bytes/delta, buf2, sizeof (buf1));
 
-		(void) printf("received %s stream in %lu seconds (%s/sec)\n",
-		    buf1, delta, buf2);
+		(void) printf("received %s stream in %llu seconds (%s/sec)\n",
```

Comment:
I would take this opportunity to make this signed rather than unsigned. At least on FreeBSD, time_t is a signed type. Conceivably the system clock may be set back during the receive and result in a negative delta, which while might be slightly more intuitive to recognize as a negative number (though not necessarily a useful number either way).

Reply:
Done